.github/workflows/publishProjectFactory.yml

```
@@ -2,76 +1,72 @@
-      run: |
-        IMAGE_TAG=egovio/project-factory-db:${{ env.BRANCH_NAME }}-${{ env.COMMIT_ID }}-${{ env.ACTION_NUMBER }}
-        docker build . \
-          --file Dockerfile \
-          --tag $IMAGE_TAG
-        echo "::set-output name=db_image_name::$IMAGE_TAG"
+  node_build:
+    runs-on: ubuntu-latest
+
+    steps:
+      # Step 1: Checkout the repository
+      - uses: actions/checkout@v3
```

actionlint (1.7.4)

the runner of "actions/checkout@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)